app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/filtersControl/filtersActionPanel/img/ic-add-filter-inline.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="20" height="20" viewBox="0 0 20 20">
```

Comment:
Please store all icons at `/src/common/img`.  And remove all unnecessary stuff from SVG file markup.